Dockerfile

```diff
@@ -1,13 +1,3 @@
-FROM node:12
-WORKDIR /app
-COPY package* ./
-RUN npm install
-COPY . .
-RUN npm run build --production
-
 FROM nginx:1.15.10-alpine
```

Comment:
Is the nginx container proxying requests?

Reply:
It does -- check `ops/nginx.conf`